### PR TITLE
Release/0.2.8

### PR DESCRIPTION
**Pull request by jtulloch**
#### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

### CODE REVIEW

File: src/components/autocomplete-dropdown/index.tsx [hunk 1]
```diff
@@ -146,11 +159,15 @@ export class AutocompleteDropdown extends React.Component<Properties, State> {
     this.close();
   };
 
-  setAnchorElements = (ref: HTMLElement) => {
+  setAnchorElements = (ref: HTMLElement): void => {
     this.anchorElement = ref;
   };
 
-  onKeyDown = (e) => {
+  // setItemsElements = (ref: HTMLElement): void => {
```
Comment (AdamDotCom) [inline src/components/autocomplete-dropdown/index.tsx:166]: Was this commented out block intended to be included?

File: src/store/zns/index.ts [hunk 2]
```diff
@@ -19,8 +18,9 @@ export interface ZnsState {
 
 const initialState: ZnsState = {
   value: {
-    route: config.defaultZnsRoute,
-    deepestVisitedRoute: config.defaultZnsRoute,
```
Comment (AdamDotCom) [inline src/store/zns/index.ts:23]: Do we still need `defaultZnsRoute` in the config file or can we remove it entirely?

File: src/components/web3-connect/index.test.tsx [hunk 3]
```diff
@@ -75,7 +149,7 @@ describe('Web3Connect', () => {
   });
 
   it('registers provider once when active is true', () => {
-    const library = { networkId: 3 };
+    const library = { networkId: Chains.Ropsten };
```
Comment (AdamDotCom) [inline src/components/web3-connect/index.test.tsx:152]: 🪄

**Review by AdamDotCom — APPROVED** [review]

File: src/store/zns/index.ts (hunk 2 above)
Comment (jtulloch) [reply src/store/zns/index.ts:23]: it's still used. just not in this spot. this default actually never even ended up getting used anyway.

**Pull request merged**